src/common-app.js

```diff
@@ -6,12 +6,14 @@ import "./lib/DOMParser"
 import commonMiddlewareSetup from "./common-middleware"
 
 import express from "express"
+import path from "path"
 import RavenServer from "raven"
 import { initializeForce } from "./current"
-import { initializeNovo } from "./novo/src/server"
+import { app as novoApp } from "./novo/src/server"
```

Comment:
Wondering yr thoughts on this @icirellik - noticed that having things wrapped in the init function was messing with reloadable. I can continue playing with it but for now just exporting app seemed to work and didn't seem to impact things.

Reply:
I do the same thing with force, I'll take a quick look.